src/graphorm/index.ts

```diff
@@ -190,15 +190,9 @@ const obj = new GraphORM({
       followingEmail: {
         transform: nullIfNotSameUser,
       },
-      followNotifications: {
-        transform: nullIfNotSameUser,
-      },
       awardEmail: {
         transform: nullIfNotSameUser,
       },
-      awardNotifications: {
```

Comment:
Check please if something in extension uses this fields, we can't break old queries.

Reply:
can you send me the code change from the frontend? 🙏